packages/gen-ai-hub/src/orchestration/orchestration-types.ts

```diff
@@ -2,40 +2,40 @@ import {
   ChatMessages,
   FilteringModuleConfig,
   InputParamsEntry,
-  LLMModuleConfig
+  LLMModuleConfig,
+  TemplatingModuleConfig
 } from './client/api/index.js';
 
 /**
- * Wrapper object to configure prompt.
+ * Prompt configuration.
  */
-export interface PromptConfig {
+export interface Prompt {
   /**
    * History.
    */
-  messages_history?: ChatMessages;
-  /**
-   * Template.
-   */
-  template: ChatMessages;
+  messagesHistory?: ChatMessages;
+
   /**
-   * Template Parameters.
+   * Template parameters.
    */
-  template_params?: Record<string, InputParamsEntry>;
+  inputParams?: Record<string, InputParamsEntry>;
 }
 
+// TODO: why do we have this extra type? and if there is a reason, why does it not apply to the filtering module?
```

Comment:
Does anyone know the answer to this question?

Reply:
For naming consistency (because the other Type and fieldname were called PromptConfig). 
The content filtering was done later and there it was missed. 

[pp] I would like if they were shorter, so without moduleConfig in the name (or at least without module, it feels unnecessary). But no strong opinion.

Reply:
But this only refers to the type, right? I would improve the names, but potentially leave the types. But also pp ;)